packages/picasso/src/SelectList/SelectList.tsx

```diff
@@ -0,0 +1,53 @@
+import MUIMenuList, { MenuListProps } from '@material-ui/core/MenuList'
+import { makeStyles, Theme } from '@material-ui/core/styles'
+import {
+  BaseProps,
+  CompoundedComponentWithRef,
+  PicassoComponentWithRef
+} from '@toptal/picasso-shared'
+import React, { forwardRef, HTMLAttributes } from 'react'
+
+import SelectListItem from '../SelectListItem'
+import styles from './styles'
+
+export type SelectListAttributes = HTMLAttributes<HTMLUListElement> &
+  Pick<MenuListProps, 'onKeyDown'>
+
+export interface Props extends BaseProps, SelectListAttributes {}
+
+export interface StaticProps {
+  Item: typeof SelectListItem
+}
+
+const useStyles = makeStyles<Theme>(styles, {
+  name: 'PicassoSelectList'
+})
+
+export const SelectList = forwardRef<HTMLUListElement, Props>(
```

Comment:
Why don't you call it `MenuList`?

```suggestion
export const MenuList = forwardRef<HTMLUListElement, Props>(
```

Reply:
@havenchyk What do you think about `MenuList` and `MenuList.Item`? Because the other component is called `DrilldownMenu`, `MenuList` seems a better option now.

Reply:
we need to brainstorm the naming :)